docs/CLI/6_reference_types.md

```diff
@@ -0,0 +1,185 @@
+# ORAS Artifact Reference Types (Alpha 1)
+
+## Pushing artifacts that reference other artifacts
+
+The focus on distributing secure supply chain artifacts has driven new innovations for supporting signatures, software bill of materials (SBoMs) and security scan results as artifacts that may be pushed, discovered and pulled, based on the tag or digest reference of a subject artifact.
+
+See the [ORAS Artifacts Spec scenarios][oras-artifacts-scenarios] for more details.
+
+## ORAS Artifact Spec Support
+
+Install the latest alpha build of the oras cli at: https://github.com/oras-project/oras/releases/tag/v0.2.1-alpha.1
+
+## Configure An Oras Artifact Enabled Registry
+
+See [Registry Support](#registry-support)
+
+## Pushing Reference Types
+
+The following walkthrough will generate the following graph of artifacts:
+
+![](../assets/images/net-monitor-graph.svg)
+
+Pushing artifact references involves identifying the unique artifact type, at least one file and the `subject` artifact being referenced.
+
+The following sample defines a new Artifact Type of **signature**, using `signature/example` as the `manifest.artifactType`.
+
+- Set environment variables for the registry and artifacts
+  ```bash
+  REPO=net-monitor
+  IMAGE=$REGISTRY/$REPO:v1
+  ARTIFACT=$REGISTRY/${REPO}:regdoc-v1
+  ```
+- Build and push an image
+
+  ```bash
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+  docker push $IMAGE
+  ```
+
+- Create a sample signature to the container image
+
+  ```bash
+  echo '{"artifact": "'${IMAGE}'", "signature": "pat hancock"}' > signature.json
+  ```
+
+- Push the signature to the registry, as a reference to the $IMAGE:
+
+  ```bash
+  oras push $REGISTRY/$REPO \
+      --artifact-type 'signature/example' \
+      --subject $IMAGE \
+      ./signature.json:application/json
+  ```
+
+## Discovering Artifact References
+
+The ORAS Artifacts Specification defines a [referrers API][oras-artifacts-referrers] for discovering references to a `subject` artifact. In the above case, oras discover can show the the list of references to the $IMAGE
+
+- Using `oras discover`, view the graph of artifacts now stored in the registry
+
+  ```bash
+  oras discover -o tree $IMAGE
+  ```
+- The output shows the beginning of a graph of artifacts, where the signature is viewed as a child of the $IMAGE
+
+  ```output
+  localhost:5000/net-monitor:v1
+  └── signature/example
+      └── sha256:1b6308bc4a2dd8933e9f66ff5bbc47e685516e5378208b46c58dc...
+  ```
+
+## Creating Deep Graphs of Artifacts
+
+The ORAS Artifacts specification enables deep graphs, enabling signed SBoMs and other artifact types.
+
+- Create and push a sample Software Bill of Materials to the registry:
+  ```bash
+  echo '{"version": "0.0.0.0", "artifact": "'${IMAGE}'", "contents": "good"}' > sbom.json
+
+  oras push $REGISTRY/$REPO \
+    --artifact-type 'sbom/example' \
+    --subject $IMAGE \
+    ./sbom.json:application/json
+  ```
+- Sign the SBoM
```

Comment:
Try using the linter in VSCode . Code fences needs a blank line following and preceding it.

Reply:
Which linter are you using? I'm not seeing warnings around indented bullets.